### PR TITLE
Updated IBFT tutorial

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated IBFT tutorial and split into three topics: 
Create IBFT network
Adding and removing validators
Create a privacy-enabled network

### CODE REVIEW

File: docs/Tutorials/Create-IBFT-Network.md [hunk 1]
```diff
@@ -4,125 +4,149 @@ description: Creating a network using IBFT consensus
...
+### 1. Install Istanbul tools 
 
-    Result:
+The [`istanbul-tools`](https://github.com/ConsenSys/istanbul-tools) repository contains tools for
+configuring IBFT networks.
 
-    ```json
-    validators
-    ```
+Install Istanbul tools.
```
Comment (NicolasMassart) [inline docs/Tutorials/Create-IBFT-Network.md:34]: I think you can remove this line

File: docs/Tutorials/Create-IBFT-Network.md [hunk 2]
```diff
@@ -4,125 +4,149 @@ description: Creating a network using IBFT consensus
...
     {
-        "Address": "0x7ae555d0f6faad7930434abdaac2274fd86ab516",
-        "Nodekey": "768b87473ba96fcfa272f958fc95a3cefdf9aa82110cde6f2f34aa5855eb39db",
-        "NodeInfo": "enode://e53e92e5a51ac2685b0406d0d3c62288b53831c3b0f492b9dc4bc40334783702cfa74c49b836efa2761edde33a3282704273b2453537b855e7a4aeadcccdb43e@0.0.0.0:30303?discport=0"
+     "Address": "0xc5db99d2cd30fd3ab58fe2738a3513c6ccdb0d2b",
+     "Nodekey": "dfdda5e2c8cedaa4468baaa46b619f5e227a7f07c4b6163edaf197991461490d",
+     "NodeInfo": "enode://7fa183662285993efaf7a59e303ec5543bbcd09cb2883e7611d9576ed90f3bcf0400b70af11c5266e5110eebe8afd4e817437bde574d686f440df1ec85822add@0.0.0.0:30303?discport=0"
     }
-
+
```
Comment (NicolasMassart) [inline docs/Tutorials/Create-IBFT-Network.md:99]: Be careful you added trailing spaces. Configure your IDE.
See markdown link errors reported in Circle CI for a lost of places where extra spaces have to be removed (19)

File: docs/Tutorials/CreatePermissionedNetwork.md [hunk 3]
```diff
@@ -84,3 +86,9 @@ existing connected nodes.
 
 However, if the connection is dropped for any reason, and a subsequent connect
 request is made from the dropped node ids, it is rejected as part of that new request.
+
+## Enhanced permissioning
+
+[Enhanced network permissioning](Enhanced/EnhancedPermissionsOverview.md) uses a smart contract permissioning
```
Comment (NicolasMassart) [inline docs/Tutorials/CreatePermissionedNetwork.md:92]: Fix this link

```suggestion
[Enhanced network permissioning](../Concepts/Permissioning/Enhanced/EnhancedPermissionsOverview.md) uses a smart contract permissioning
```

**Review by NicolasMassart — APPROVED** [review]
LGTM (but still requires QA checks to pass).

File: docs/Tutorials/Create-IBFT-Network.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Tutorials/Create-IBFT-Network.md:34]: Done

File: docs/Tutorials/CreatePermissionedNetwork.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Tutorials/CreatePermissionedNetwork.md:92]: Fixed

**Pull request merged**